wg-meetings/2020-03-25.md

```diff
@@ -0,0 +1,94 @@
+# Node.js  Diagnostics WorkGroup Meeting 2020-03-25
+
+## Links
+
+* **Recording**:  
+* **GitHub Issue**: https://github.com/nodejs/diagnostics/issues/367
+* **Minutes Google Doc**: https://docs.google.com/document/d/1DAss-D2v2zlblkc9fPaX9vNjd31yS9Z1vf5I3vI54sM/edit
+* **Recording**: Unavailable
```

Comment:
There are two **Recording** sections.   @gireeshpunathil we should also make sure to get you setup so you can help stream/record.  Have we gotten you access to the community Zoom and made you a YouTube manager?

Reply:
@mhdawson - thanks, fixed.

re: zoom + youtube manager, yes. I just need someone to teach me how to use those. (will catchup with you offline)